exporter/chronicleexporter/README.md

```diff
@@ -22,18 +22,19 @@ This exporter facilitates the sending of logs to Chronicle, which is a security
-| `compression`       | string            | `none`                                         | `false`  | The compression type to use when sending logs. valid values are `none` and `gzip`           |
-| `ingestion_labels`  | map[string]string |                                                | `false`  | Key-value pairs of labels to be applied to the logs when sent to chronicle.                 |
+| Field                  | Type              | Default                                | Required | Description                                                                                 |
+| ---------------------- | ----------------- | -------------------------------------- | -------- | ------------------------------------------------------------------------------------------- |
+| `endpoint`             | string            | `malachiteingestion-pa.googleapis.com` | `false`  | The Endpoint for sending to chronicle.                                                      |
+| `creds_file_path`      | string            |                                        | `true`   | The file path to the Google credentials JSON file.                                          |
+| `creds`                | string            |                                        | `true`   | The Google credentials JSON.                                                                |
+| `log_type`             | string            |                                        | `true`   | The type of log that will be sent.                                                          |
+| `raw_log_field`        | string            |                                        | `false`  | The field name for raw logs.                                                                |
+| `customer_id`          | string            |                                        | `false`  | The customer ID used for sending logs.                                                      |
+| `override_log_type`    | bool              | `true`                                 | `false`  | Whether or not to override the `log_type` in the config with `attributes["log_type"]`       |
+| `namespace`            | string            |                                        | `false`  | User-configured environment namespace to identify the data domain the logs originated from. |
+| `compression`          | string            | `none`                                 | `false`  | The compression type to use when sending logs. valid values are `none` and `gzip`           |
+| `ingestion_labels`     | map[string]string |                                        | `false`  | Key-value pairs of labels to be applied to the logs when sent to chronicle.                 |
+| `collect_host_metrics` | bool              | `true`                                 | `false`  | Enables collecting metrics about the collector's process and log ingestion metrics          |
```

Comment:
About the naming, should we have this be something like `collect_agent_metrics` instead of `collect_host_metrics`? It seems like the scope is just metrics about the agent specifically, and not so much about the host. (e.g. how much cpu time the agent is taking, # of logs the agent has sent, etc)